app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/AcceptOfferDialog.scala

```diff
@@ -224,7 +242,16 @@ class AcceptOfferDialog {
         val offerHex = offerTLVTF.text.value
         val offer = LnMessageFactory(DLCOfferTLV).fromHex(offerHex)
 
-        Some(AcceptDLCOffer(offer))
+        if (peerAddressTF.text.value.nonEmpty) {
+          val peer = new URI("tcp://" + peerAddressTF.text.value)
```

Comment:
It'd be great to make port optional for this field, so the end user could type just an onion address without port.

```scala
val peer = new URI("tcp://" + NetworkUtil.parseInetSocketAddress(peerAddressTF.text.value, DefaultDLCPort))```